src/styles/App.css

```diff
@@ -1,6 +0,0 @@
-.App {
-  align-content: center;
-  margin: auto;
-  width: 60%;
-  background-image: url("/public/images/bg/08.jpg");
```

Comment:
when use use `background-image` then don't forget to use the `background-size` attribute which will help you size the image in the way you want.

For example, if you wanted it to stretch to fill the entire container, you'd add:
`background-size: cover;`